### PR TITLE
Add track function and update page function.

### PR DESCRIPTION
**Pull request by rayd**
Appcues now supports a track() function and a fuller page() implementation.

### CODE REVIEW

File: test/index.test.js [hunk 1]
```diff
@@ -76,13 +76,23 @@ describe('Appcues', function() {
...
+        analytics.stub(window.Appcues, 'page');
       });
 
-      it('should proxy to Appcues.start()', function() {
-        analytics.didNotCall(window.Appcues.start);
-        analytics.page('Pricing');
-        analytics.called(window.Appcues.start);
+      it('should call Appcues.page', function() {
+        analytics.page('some page', { someAttr: true });
+        analytics.called(window.Appcues.page, 'some page');
```
Comment (f2prateek) [inline test/index.test.js:84]: This should verify `someAttr: true` is sent as well.

**Comment by rayd** [comment]
Added an assertion to the test to verify that `someAttr` is being passed. There's a handful of other properties that are passed along in the second arg (by `analytics.page`) and I didn't want to make assumptions about what those might be, so I'm directly checking the args property of the `Appcues.page` spy for the `someAttr` value. Does that seem like the right approach?

**Comment by rayd** [comment]
Anything we can do on our side to help move this along? Thanks!

**Comment by hijonathan** [comment]
@f2prateek any update for us here? We have a few mutual customers who are patiently waiting for this fix.

**Comment by f2prateek** [comment]
@rayd @hijonathan sorry I didn't realize my initial comment had been addressed. LGTM.

**Comment by SeedSultan** [comment]
We're one of the customers waiting for a fix 👍 Look forward to seeing this resolved!

**Pull request merged**